workers/unity/Packages/io.improbable.gdk.testutils/Editor/Tests/SpatialDeploymentManagerTests.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Linq;
+using System.Net;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.TestUtils.Editor.Tests
+{
+    [TestFixture]
+    public class SpatialDeploymentManagerTests
+    {
+        [Test]
+        public void Start_throws_an_exception_if_deployment_config_does_not_exist()
```

Comment:
These first 2 tests are a bit generic in their error checking. What happens if things work, but an unrelated exception throws?

Reply:
Discussed offline - the alternative is to assert against the message of the exception, which is undesirable